### ISSUE TITLE
AWS ELB modules reference non-existent certs

### ISSUE THREAD

**Issue by siddharthist** [CiscoCloud/mantl#1688]
This is from runnning `terraform plan` with the default (completely unchanged) `aws.sample.tf`:
```
Errors:
  * file: open ./ssl/certs/host.cert.pem: no such file or directory in:
${file(var.ssl_cert_file)}
  * file: open ./ssl/certs/host.cert.pem: no such file or directory in:
${file(var.ssl_cert_file)}
```
```
terraform/aws/elb/main.tf
1:variable ssl_cert_file { default = "./ssl/certs/host.cert.pem" }

terraform/aws/elb/traefik/main.tf
1:variable ssl_cert_file { default = "./ssl/certs/host.cert.pem" }
```
- Ansible version (`ansible --version`): 1.9.4
- Python version (`python --version`): 2.7.11
- Git commit hash or branch: master
- Cloud Environment: all
- Terraform version (`terraform version`): 0.7.0

**Comment by ryane** [comment]
this broke when we moved the cert generation to the hosts (#1463, I think) and we appear to have missed it b/c

1. we are testing with the legacy terraform file (#1693)
2. existing certs from pre #1463 still in our local directory (this was the case for me)

I'm not sure how to fix this now that the certs are generated post-terraform and on the hosts. Anybody have an idea? Do we just remove the elbs altogether?

**Comment by siddharthist** [comment]
@ryane We could remove the ELBs, or pull the certs from the edge hosts using Ansible's `fetch` module.

**Comment by ryane** [comment]
not sure I understand how the second option would work. It seems like the hosts would not be available when terraform provisions the elbs.

**Comment by siddharthist** [comment]
@ryane You're totally right I don't know what I was thinking :)

**Comment by snoby** [comment]
@ryane @siddharthist @stevendborrelli 
This ticket and the decision to remote the ELB's from ticket #1710.  Puts myself as a user of mantl into a tough spot.  We deployed V1.2 which still has the ELB's in place.  However there is no upgrade path for V1.3 on AWS.  
There is no addon for me to enable ELBs.  What do I do now?  Do I fork the repository and just stick on v1.2? Do I hand port in changes?

Having an ELB front the edge nodes is a good thing because we get health checks, also the ability to have those ELB's do SSL termination if we want.

@johntdyer -Might have input to this as well.

Basically I'm asking let's find a way to not leave AWS users hanging.  How can we work together to solve this?
Thanks,
